skonaki.py

```diff
@@ -81,8 +81,23 @@ def main():
         help="Output format, choose between: text (default), json",
         default="text",
     )
+    parser.add_argument(
+        "--run-whisper-locally",
+        help="Run the Whisper API locally (default: False)",
+        required=False,
+        action="store_true"
+    )
     args = parser.parse_args()
 
+    if args.run_whisper_locally:
+        try:
+            global whisper
```

Comment:
Why do we need this?

Reply:
The global part is there so that the whisper library can be used in the `generate_summary()` function. 

I opted for this as it felt closer to importing a library in the "traditional" way, however another approach could be to remove the `global whisper` and pass it as a parameter instead to generate_summary(). What do you think?

Reply:
Right. OK, then this looks error prone if someone calls `generate_summary` directly, e.g. from their own module (e.g. a GUI) or from unit tests. Let's move the import logic into the `if` branch that needs it inside `generate_summary` (i.e. around L180). Then there shall be no need for `global`.

Reply:
This adjustment should now be implemented :rocket: